python/cudf/cudf/tests/test_string.py

```diff
@@ -971,3 +971,119 @@ def test_string_no_children_properties():
 
     assert empty_col._nbytes == 0
     assert getsizeof(empty_col) >= 0  # Accounts for Python GC overhead
+
+
+@pytest.mark.parametrize(
+    "string",
+    [
+        ["Cbe", "cbe", "CbeD", "Cb", "ghi", "Cb"],
+        ["abc", "xyz", "a", "ab", "123", "097"],
+        ["abcdefghij", "0123456789", "9876543210", None, "accénted", ""],
+    ],
+)
+@pytest.mark.parametrize(
+    "index", [0, 1, 2, 3, 9, 10],
+)
+def test_string_get(string, index):
+    pds = pd.Series(string)
+    gds = Series(string)
+
+    assert_eq(pds.str.get(index).fillna(""), gds.str.get(index).fillna(""))
+
+
+# @pytest.mark.parametrize(
```

Comment:
Had to disable tests as well due to the signature conflict with nvstrings as nvstrings mandates `start` param, it should be an optional param. Will enable these tests when I enable the string methods plumbings.